le.go

```diff
@@ -43,9 +45,10 @@ const maxLogLength int = 65000 //add 535 chars of headroom for the filename, tim
 // choosing manual configuration and token based TCP connection.
 func Connect(host, token string) (*Logger, error) {
 	logger := Logger{
-		host:          host,
-		token:         token,
-		lastRefreshAt: time.Now(),
+		host:              host,
+		token:             token,
+		lastRefreshAt:     time.Now(),
+		_testWaitForWrite: nil,
```

Comment:
This is the default, doesn't need to be specified.

Reply:
👍